engine/exec/exec.go

```diff
@@ -27,6 +32,16 @@ func Run(ctx context.Context, step *spec.Step, output io.Writer) (*runtime.State
 	cmd.Env = toEnv(step.Envs)
 	cmd.Stderr = output
 	cmd.Stdout = output
+	if !isDrone && len(step.Command) > 0 {
+		stepCommand := strings.TrimSpace(strings.Join(step.Command, ""))
+		if stepCommand != "" {
+			printCommand := "Executing the following command(s):\n" + stepCommand
+			lines := strings.Split(printCommand, "\n")
```

Comment:
Splitting on '/n' isn't going to work for all cases.

Reply:
that's what i thought, but it was working, i tried with putting \n, \\n in the command